### PR TITLE
Mint clone feature branch

### PR DESCRIPTION
**Pull request by silverside3000**

### CODE REVIEW

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review by: Inyama, Uchechukwu Henry.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hello team! you have made some progress :smiley: 

### However, there are changes to be implemented.

- [x] There are no linter checks, on your project. Please, implement the necessary changes and ensure it passes the test.
- [x] Validate your html file to ensure there are no errors use this link [**Html Validator**](https://validator.w3.org/)

[**observation**] when you hold the left key of your mouse and hover your design, the webpage moves around with the cursor.

[**Observation**] There's an Html syntax inside your email address, fix that.
- [x] The sign in button is not properly aligned.



### Implement the changes, linter update and Html validation, then submit another form for code review. :fire:

**Comment by samgaco** [comment]
Reviewing...

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+    config: 'stylelint.config.js'
```
Comment (samgaco) [inline .stickler.yml:3]: :+1:

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @mosesmuchemi and @silverside3000 :high_brightness: 

All the changes required by the previous TSE have been implemented correctly. Anyways still something to address:

1. A horizontal scrolling bar is showing, try to make it disappear.
2. Try to have your styling looking closer to the one in the original page, colors, images, alignment of texts, etc.

------------------------

Have into account moving on :arrow_down: 

* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

---------------------
* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by uche-inyama — COMMENTED** [review]
## Pull Request under review by: Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi @team :raised_hand_with_fingers_splayed: nice job so far,

### But there are changes that were not implemented

- [ ]  Your sign-in button hasn't been properly positioned. **Previous request from @uche-inyama**
- [ ] Try to have your styling looking closer to the one in the original page, colors, images, alignment of texts, etc. **Previous request from @samgaco**

- [ ] Also, use descriptive comments in your commits, not this **fixing one more annoying lint error** :pray:  

### After adding these changes submit another form for code review. :muscle: 

Your project hasn't been approved :stop_sign:

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi team :wave: made much improvement.

### But your button is not positioned properly, kindly do so, here is a snap shot of what it looks like.
Screenshot from 2019-09-19 02-35-40

- [ ] Look up how to implement a  line with **mark down** in google and use it to implement the link in your README.md file showing in your master branch; It would make it look neat :smiley: 
### Kindly make these adjustments and submit another form for code review.

## This project is not approved. :stop_sign:

**Comment by silverside3000** [comment]
@uche-inyama  for the life of me i don't what web browser you are viewing this project from but I assure you that it does looks good on chrome, mozila, edge, opera and uc browsers and so asking us to go redo this cos of a web browser defect is very undeserving punishment cos we are just going to abandon it, tell me how do you fix something that isn't broken from your point of view?

edge mint
opera mint
uc mint
chrome mint
Mozilla mint

**Comment by mosesmuchemi** [comment]
everything seems fine now

File: intuit.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
+.body_style {
+  background-color: #dfdfdf;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.sign-in-part {
+  width: 23%;
```
Comment (maelfosso) [inline intuit.css]: Please remove that `width: 23%` to fix the error @uche-inyama mentioned above.

File: intuit.css [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  padding-top: 100px;
+  font-size: smaller;
+  width: 55%;
+  padding-bottom: 20px;
+}
+
+.top_image {
+  padding-bottom: -20px;
+  padding-left: 40px;
+}
```
Comment (maelfosso) [inline intuit.css:32]: Remove these two properties please 
```css
padding-bottom: -20px;
padding-left: 40px;
```

Try something like
```css
.top_image {
  display: flex;
  margin: auto;
  justify-content: space-around;
}
```

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello @mosesmuchemi  and @silverside3000 

Thanks for that job.

I left comment below that could help you to fix the errors @uche-inyama  detected.
Please, based on that comment, fix the errors and submit a new code review request.
Thanks.

File: intuit.css (hunk 2 above)
Comment (mosesmuchemi) [reply intuit.css]: removed the width: 23% property

File: intuit.css (hunk 3 above)
Comment (mosesmuchemi) [reply intuit.css:32]: changed the top_image class property as per the recommendations

**Review by wathigo — APPROVED** [review]
## Hi devs
### Great effort so far, :clap: 
**You may move on :)**
_Good luck in your future projects :)_

**Pull request merged**